### ISSUE TITLE
Isn't really an issue, but...

### ISSUE THREAD

**Issue by Didys-pog** [Xemorr/superheroes#18]
Would be really cool if you could mix up things, for example when shifting a special kind of skill happens, for example aura.

**Comment by Xemorr** [comment]
Yes, I need to write a standard set of conditions for all skills.
